examples/app.py

```diff
@@ -85,7 +88,7 @@
     CELERY_EAGER_PROPAGATES_EXCEPTIONS=True,
     CELERY_RESULT_BACKEND="cache",
     DEBUG=True,
-    SEARCH_UI_SEARCH_API='invenio_search_ui.api',
+    SEARCH_UI_SEARCH_API='invenio_records_rest.recid_list',
```

Comment:
@jirikuncar Then the two apps needs to know about each other and run in the same process. I.e. you won't be able to run the UI app on set of servers and the API app on another set of servers.

I think it's better to just point to a URL endpoint. Then you can support both e.g. local (``/api/records/``) and remote (``https://api.example.org/records/``) endpoints.